modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1519,6 +1559,17 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     return Bluebird.reject(new Error('AbtractUtxoCoin method not implemented'));
   }
 
+  /**
+   * The full name of the coin we should use while making quries to blockchair
+   * Blockchair uses slightly different naming conventions for some of the utxo coins
+   * Hence, some utxo coins should override this function and output the accurate
+   * string
+   * @returns stringg
```

Comment:
```suggestion
   * @returns {string}
```

Jsdoc types should be enclosed in braces: `{string}`